src/sentry/api/endpoints/organization_group_index.py

```diff
@@ -320,7 +321,7 @@ def put(self, request, organization):
             projects,
             self.get_environments(request, organization),
         )
-        return update_groups(request, projects, organization.id, search_fn)
+        return update_groups(request, projects, organization.id, search_fn, has_inbox)
```

Comment:
I'm more tempted to have the has_inbox declaration inside of update_groups rather than changing the function signature, but this might be more of a nit than any sort of blocker, especially given the temporary nature of the flag.